{{cookiecutter.project_slug}}/clients/mobile/react-native/eas.json

```diff
@@ -12,7 +12,6 @@
         "BACKEND_SERVER_URL": "<REPLACE_WITH_LOCAL_BACKEND_SERVER_URL>",
         "BUILD_ENV": "development",
```

Comment:
This isnt ever used anywhere, it may have been being used as part of `npx run` commands but is not necessary since the build command will inject the variables in the .env